src/main/java/crazypants/enderio/material/OreDictionaryPreferences.java

```diff
@@ -62,7 +62,7 @@ public ItemStack getPreferred(ItemStack stack) {
     }
 
     if(result == null) {
-      result = stack;
+      result = stack.copy();
     }
     stackCache.put(key, result);
     return stack;
```

Comment:
I though about it, but it seems more prudent to always expect to get a "dirty" stack from any method---Forege and MC methods all do that.